src/lib/plugins/players.js

```diff
@@ -32,7 +37,7 @@ module.exports.server = function (serv, { version }) {
         else throw new UserError('Console cannot set gamemode itself')
       }
 
-      return str.match(/^(survival|creative|adventure|spectator|[0-3]) (\w+)$/) || false
+      return str.match(/^(survival|creative|adventure|spectator|[0-3])( @[arspe](?:\[([^\]]+)\])?| \w+)?$/) || false
       // return params || false
```

Comment:
Remove this comment

Reply:
Why? I think having a descriptive comment of the cryptic regex is useful.

Reply:
Ok